AmericanMedicalAssociation/ama-style-guide-2
# EWL-5810 global header (merge before PR #439  and #440)

## JIRA Ticket(s)
- [EWL-5810: A1 | Theme Global Header](https://issues.ama-assn.org/browse/EWL-5810)


## Description:
Finish theming the global header

## To Test:
- visit http://localhost:3000/?p=organisms-main-navigation-as-non-member
- click on the usermenu link that says `John Smithe`
- observe the page doesn't reload or jump to the top

## Relevant Screenshots/GIFs:
<img width="1359" alt="screen shot 2018-10-10 at 3 29 20 pm" src="https://user-images.githubusercontent.com/2271747/46764136-47ba6900-cca1-11e8-80fe-36a77375bb5a.png">


## Additional Notes:
this PR is a dependency of https://github.com/AmericanMedicalAssociation/ama-d8/pull/875

### This PR is the originator of both #439 and #440. I branched off of this PR to make both of those PRs and needs to be merged before both of thos PRs
---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
